File: 프로그래머스/unrated/181906. 접두사인지 확인하기/접두사인지 확인하기.java
class Solution {
    public int solution(String my_string, String is_prefix) {
        String str = "";
        
        for (int i = 0; i < my_string.length(); i++) {
            str += my_string.charAt(i);
            
            if (str.equals(is_prefix)) {
                return 1;
            }
        }
        
        return 0;
    }
}

File: 프로그래머스/unrated/181906. 접두사인지 확인하기/접두사인지 확인하기.js
function solution(my_string, is_prefix) {
    if (my_string.length < is_prefix.length) {
        return 0;
    }
    
    let sum = '';
    for (let i = 0; i < my_string.length; i++) {
        sum += my_string[i];
        
        if (sum === is_prefix) {
            return 1;
        }
    }
    return 0;
}